src/services/database.ts

```diff
@@ -93,3 +102,28 @@ const CriteriaIndexMap: Record<Exclude<SearchCriteria, SearchCriteria.ALL>, Tabl
   vin: 'VinIndex',
   trailerId: 'TrailerIdIndex',
 };
+export const postTechRecord = async (request: any) => {
+  if (process.env.AWS_SAM_LOCAL) {
+    return '123';
+  }
+  const command = {
+    TableName: tableName,
+    ConditionExpression: '#vin <> :vin AND #systemNumber <> :systemNumber',
+    ExpressionAttributeNames: {
+      '#vin': 'vin',
+      '#systemNumber': 'systemNumber',
+    },
+    ExpressionAttributeValues: {
+      ':vin': { S: request.vin },
+      ':systemNumber': { S: request.systemNumber },
+    },
+    Item: marshall(request as Record<string, AttributeValue>),
+  };
+  try {
+    return await ddbClient.send(new PutItemCommand(command));
```

Comment:
```suggestion
    return ddbClient.send(new PutItemCommand(command));
```
return await should be avoided (async functions returns a promise anyways, it can be awaited where the function is called)

Reply:
One more comment on here, what does this actually return?? Are we returning the record back? Is it just a success? If it is the record back, should this not be unmarshalled?

Reply:
The record doesn't come pack in the payload as standard so it isn't currently being returned. Looking at the spec, you can set ReturnValues to return the "updated record" - happy to return it upstream

Reply:
You could just do something like this:
```ts
await ddbClient.send(new PutItemCommand(command))
return request // <- as TechRecord

```